internal/entclient/request.go

```diff
@@ -8,6 +8,9 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// request() makes an HTTP request of the specified method, returning the
+// *http.Response and any error.
+
```

Comment:
There shouldn't be an empty line between the comment and the function.

Reply:
I did this on purpose for the two in this file, mostly as comments to myself or someone like me first seeing the code. I left the blank line so go doc wouldn't attach to the funcs (and go linters wouldn't complain about documenting a private func).

But I can just remove the comments if you like.